src/epggrab/module/eit.c

```diff
@@ -391,12 +391,21 @@ static int _eit_desc_component
 static int _eit_desc_content
   ( epggrab_module_t *mod, const uint8_t *ptr, int len, eit_event_t *ev )
 {
+  uint8_t tempPtr = 0;  //Temporary variable to hold a (potentially) changed *prt value.
```

Comment:
"... changed *prt value": You mean *ptr?

Reply:
Done